docs/syntax/assignments.md

```diff
@@ -49,7 +90,7 @@ echo(x) # Error: x is not defined
 ```
 
 Worth to note that if a variable gets re-defined within these expressions,
-it will temporarely assume its new value, but will rollback to the original
+it will temporaraly assume its new value, but will rollback to the original
```

Comment:
```suggestion
it will temporarily assume its new value, but will rollback to the original
```

Reply:
Spelling is a lost art (on me anyway) :-).
I will fix this.